 Network rafaelpadovezi-django-orm_default  Creating
 Network rafaelpadovezi-django-orm_default  Created
 Container rafaelpadovezi-django-orm-db-1  Creating
 Container rafaelpadovezi-django-orm-db-1  Created
 Container rafaelpadovezi-django-orm-api2-1  Creating
 Container rafaelpadovezi-django-orm-api1-1  Creating
 Container rafaelpadovezi-django-orm-api2-1  Created
 Container rafaelpadovezi-django-orm-api1-1  Created
 Container rafaelpadovezi-django-orm-nginx-1  Creating
 Container rafaelpadovezi-django-orm-nginx-1  Created
Attaching to api1-1, api2-1, db-1, nginx-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
db-1     | selecting default time zone ... Etc/UTC
db-1     | creating configuration files ... ok
db-1     | running bootstrap script ... ok
db-1     | performing post-bootstrap initialization ... ok
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | waiting for server to start....2024-03-11 02:27:56.187 GMT [48] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-03-11 02:27:56.189 GMT [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-03-11 02:27:56.201 GMT [51] LOG:  database system was shut down at 2024-03-11 02:27:55 GMT
db-1     | 2024-03-11 02:27:56.227 GMT [48] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/rinha.dump.sql
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     |  set_config 
db-1     | ------------
db-1     |  
db-1     | (1 row)
db-1     | 
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | SET
db-1     | CREATE TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | CREATE TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | CREATE TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | CREATE TABLE
db-1     | ALTER TABLE
db-1     | CREATE TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | CREATE TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | CREATE TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | CREATE TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | CREATE TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | CREATE TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | COPY 0
db-1     | COPY 0
db-1     | COPY 24
db-1     | COPY 0
db-1     | COPY 0
db-1     | COPY 0
db-1     | COPY 5
db-1     | COPY 0
db-1     | COPY 6
db-1     | COPY 15
db-1     |  setval 
db-1     | --------
db-1     |       1
db-1     | (1 row)
db-1     | 
db-1     |  setval 
db-1     | --------
db-1     |       1
db-1     | (1 row)
db-1     | 
db-1     |  setval 
db-1     | --------
db-1     |      24
db-1     | (1 row)
db-1     | 
db-1     |  setval 
db-1     | --------
db-1     |       1
db-1     | (1 row)
db-1     | 
db-1     |  setval 
db-1     | --------
db-1     |       1
db-1     | (1 row)
db-1     | 
db-1     |  setval 
db-1     | --------
db-1     |       1
db-1     | (1 row)
db-1     | 
db-1     |  setval 
db-1     | --------
db-1     |       1
db-1     | (1 row)
db-1     | 
db-1     |  setval 
db-1     | --------
db-1     |       1
db-1     | (1 row)
db-1     | 
db-1     |  setval 
db-1     | --------
db-1     |       6
db-1     | (1 row)
db-1     | 
db-1     |  setval 
db-1     | --------
db-1     |      15
db-1     | (1 row)
db-1     | 
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | CREATE INDEX
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | ALTER TABLE
db-1     | 
db-1     | 
db-1     | waiting for server to shut down...2024-03-11 02:27:56.946 GMT [48] LOG:  received fast shutdown request
db-1     | .2024-03-11 02:27:56.948 GMT [48] LOG:  aborting any active transactions
db-1     | 2024-03-11 02:27:56.950 GMT [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
db-1     | 2024-03-11 02:27:56.951 GMT [49] LOG:  shutting down
db-1     | 2024-03-11 02:27:56.953 GMT [49] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-03-11 02:27:57.520 GMT [49] LOG:  checkpoint complete: wrote 959 buffers (5.9%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.291 s, sync=0.243 s, total=0.570 s; sync files=352, longest=0.208 s, average=0.001 s; distance=4495 kB, estimate=4495 kB; lsn=0/194E6D0, redo lsn=0/194E6D0
db-1     | 2024-03-11 02:27:57.527 GMT [48] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-03-11 02:27:57.571 GMT [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-03-11 02:27:57.571 GMT [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-03-11 02:27:57.571 GMT [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-03-11 02:27:57.577 GMT [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-03-11 02:27:57.584 GMT [73] LOG:  database system was shut down at 2024-03-11 02:27:57 GMT
db-1     | 2024-03-11 02:27:57.591 GMT [1] LOG:  database system is ready to accept connections
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
nginx-1  | 2024/03/11 02:27:59 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.112.4:8080/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/03/11 02:27:59 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.112.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.112.3:8080/clientes/1/extrato", host: "localhost:9999"
api1-1   | [2024-03-11 02:28:00 +0000] [1] [INFO] Starting gunicorn 21.2.0
api1-1   | [2024-03-11 02:28:00 +0000] [1] [INFO] Listening at: http://0.0.0.0:8080 (1)
api1-1   | [2024-03-11 02:28:00 +0000] [1] [INFO] Using worker: sync
api1-1   | [2024-03-11 02:28:00 +0000] [8] [INFO] Booting worker with pid: 8
api2-1   | [2024-03-11 02:28:00 +0000] [1] [INFO] Starting gunicorn 21.2.0
api2-1   | [2024-03-11 02:28:00 +0000] [1] [INFO] Listening at: http://0.0.0.0:8080 (1)
api2-1   | [2024-03-11 02:28:00 +0000] [1] [INFO] Using worker: sync
api2-1   | [2024-03-11 02:28:00 +0000] [8] [INFO] Booting worker with pid: 8
api2-1   | [2024-03-11 02:28:00 +0000] [9] [INFO] Booting worker with pid: 9
api1-1   | [2024-03-11 02:28:00 +0000] [9] [INFO] Booting worker with pid: 9
nginx-1  | 2024/03/11 02:28:01 [error] 30#30: *4 no live upstreams while connecting to upstream, client: 192.168.112.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/03/11 02:28:03 [error] 30#30: *5 no live upstreams while connecting to upstream, client: 192.168.112.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/03/11 02:28:05 [error] 30#30: *6 no live upstreams while connecting to upstream, client: 192.168.112.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/03/11 02:28:07 [error] 30#30: *7 no live upstreams while connecting to upstream, client: 192.168.112.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/03/11 02:28:09 [error] 30#30: *8 no live upstreams while connecting to upstream, client: 192.168.112.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1 exited with code 0
nginx-1 exited with code 0
api2-1   | [2024-03-11 02:32:24 +0000] [1] [INFO] Handling signal: term
api1-1   | [2024-03-11 02:32:24 +0000] [1] [INFO] Handling signal: term
api2-1   | [2024-03-11 02:32:24 +0000] [8] [INFO] Worker exiting (pid: 8)
api1-1   | [2024-03-11 02:32:24 +0000] [9] [INFO] Worker exiting (pid: 9)
api1-1   | [2024-03-11 02:32:24 +0000] [8] [INFO] Worker exiting (pid: 8)
api2-1   | [2024-03-11 02:32:24 +0000] [9] [INFO] Worker exiting (pid: 9)
api1-1   | [2024-03-11 02:32:24 +0000] [1] [INFO] Shutting down: Master
api2-1   | [2024-03-11 02:32:24 +0000] [1] [INFO] Shutting down: Master
api1-1 exited with code 0
api1-1 exited with code 0
api2-1 exited with code 0
api2-1 exited with code 0
db-1     | 2024-03-11 02:32:25.929 GMT [1] LOG:  received fast shutdown request
db-1     | 2024-03-11 02:32:25.933 GMT [1] LOG:  aborting any active transactions
db-1     | 2024-03-11 02:32:25.936 GMT [1] LOG:  background worker "logical replication launcher" (PID 76) exited with exit code 1
db-1     | 2024-03-11 02:32:25.936 GMT [71] LOG:  shutting down
db-1     | 2024-03-11 02:32:25.939 GMT [71] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-03-11 02:32:26.090 GMT [71] LOG:  checkpoint complete: wrote 660 buffers (4.0%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.009 s, sync=0.120 s, total=0.155 s; sync files=29, longest=0.063 s, average=0.005 s; distance=19713 kB, estimate=19713 kB; lsn=0/2C8EC60, redo lsn=0/2C8EC60
db-1     | 2024-03-11 02:32:26.095 GMT [1] LOG:  database system is shut down
db-1 exited with code 0
